jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/AdministrationControllerTest.groovy

```diff
@@ -159,4 +161,27 @@ class AdministrationControllerTest extends Specification {
         where: 'notEditable - list of predefined group names'
             notEditable << [ADMIN, USER, BANNED_USER]
     }
+
+    def 'user in group should paginated with 20 per page'() {
+        given: 'User logged in and has admin rights'
+            def session = users.signInAsAdmin(forum)
+        and: 'Group created'
+            def groupDto = Groups.randomDto()
+            groups.create(groupDto, session)
+        and: 'Users in group created'
+            for (int i=0; i<45; i++) {
+                users.createdWithoutAccess(new User()).withGroups([groupDto.name])
+            }
+            def group = groupsService.getGroupByName(groupDto.name)
+            def groupUserList = group.users
+            Collections.sort(groupUserList, new GroupsService.UserByNameComparator())
+        when: 'User attempts to retrive paginated list of users in group'
+            MvcResult page1 = groups.getPagedGroupUsers(group.id, 1L, session)
```

Comment:
Only group id is used inside to build http request string and most part of methods in Groups module use `long groupId` as a parameter.